docs/content/chapter_3.md

```diff
@@ -0,0 +1,85 @@
+# Project Environment
+
+## Introduction
+
+In here we'll discuss about the project environment. We'll discuss about the project architecture & folder structure, project environments (`Production`, `Development`, `Staging`), how secrets `(environment variables)` are handled, how to use environment variables for different environments,
```

Comment:
It's better if we can write this paragraph as an unordered list.

**Here we'll discuss,** 

- **Project environment**
- **Project architecture & folder structure**
- **Project environments (`Production`, `Development`, `Staging`)**
- **How secrets `(environment variables)` are handled**
- **How to use environment variables for different environments**

Reply:
done